extensions/better-deal/package.json

```
@@ -0,0 +1,65 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "better-deal",
+  "title": "Better Deal",
+  "description": "Calculates unit costs and tracks purchasing history to help you find the best deals.",
+  "icon": "extension_icon.png",
+  "author": "juxtdesigncc",
+  "categories": [
+    "Productivity",
+    "Finance"
+  ],
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "index",
+      "title": "Add Better Deal",
+      "description": "Add new deal to compare",
+      "mode": "view",
+      "arguments": [
+        {
+          "name": "price",
+          "type": "text",
+          "placeholder": "Cost",
+          "required": true
+        },
+        {
+          "name": "quantity",
+          "type": "text",
+          "placeholder": "Quantity",
+          "required": true
+        },
+        {
+          "name": "unitSize",
+          "type": "text",
+          "placeholder": "Unit Size",
+          "required": false
+        }
+      ]
+    },
+    {
+      "name": "view",
+      "title": "View Better Deal",
```

```suggestion
      "title": "View Better Deal History",
```
I would also add a [List.EmptyView](https://developers.raycast.com/api-reference/user-interface/list#list.emptyview) that tells what the command is for, I was confused at first